scripts/zones/Pashhow_Marshlands/mobs/Ni_Zho_Bladebender.lua

```diff
@@ -1,6 +1,6 @@
 -----------------------------------
 -- Area: Pashhow Marshlands
---  MOB: NiZho Bladebender
```

Comment:
Is he Ni_Zho_Bladebender in the sql? normally we don't convert apostrophes to spaces, we strip them.

Reply:
my preference would be correcting the sql name

Reply:
I see boohoo has the same naming convention you used, man the inconsistencies in the project annoy me sometimes.

Reply:
...nvm me, I'm just gonna mass correct them all later.